### PR TITLE
feat(feedback): auto-file GitHub issues from classified feedback with dedup/clustering (closes #498)

### LINKED ISSUE #498: feat(feedback): auto-file GitHub issues from classified feedback with dedup/clustering
## Why
Turns classified feedback into pipeline-ready issues and prevents duplicate spam (docs/launch-and-marketing-plan.md (PR #495) Section B.2/B.3).

## Scope
- New `utilities/feedback/issue_service.py`: create GitHub issues (gh/API) with a RUNBOOK MODE=start-shaped body (Why/Scope/Files/Acceptance), always adding `agent:ready` when risk=none & confidence≥0.7; adding the matching `risk:*`+`needs-human`+Decision Comment otherwise. Stamp issue number back on the feedback row and reference the plan doc.
- Embedding-based dedup: before filing, cosine-compare to open clusters; if similar, +1 comment on the existing issue instead of opening a new one; nightly `auto_recluster_feedback` beat task.
- Abuse guards: per-user rate limit; feature clusters need a demand threshold before agent:ready.

## Files
`utilities/feedback/issue_service.py`, a beat entry in `app/my_celery.py`, unit tests.

## Acceptance
- Dedup + label logic unit-tested; issue body matches MODE=start expectations; ≥80% patch coverage.

### PR DESCRIPTION
**Pull request by gitchrisqueen**
## What & why

Stage 3 of the feedback→auto-work loop (`docs/launch-and-marketing-plan.md` §B.2/B.3). Stage 1 (#496) captures feedback, stage 2 (#497) classifies it; this stage turns an `auto_work` verdict into a **pipeline-ready GitHub issue** — and, crucially, prevents the same recurring problem from becoming N issues.

### New `utilities/feedback/issue_service.py`

- **Issue bodies are shaped for the RUNBOOK's `MODE=start`** — `## Why` / `## Scope` / `## Files` / `## Acceptance`, with a per-component files hint, the classifier verdict, and a reference to the plan doc. Title is conventional-commit shaped (`fix(feed-commenting): …`).
- **Label logic** (plan §B.2): the classifier's taxonomy labels + `feedback-loop`, then **exactly one** of
  - `agent:ready` — risk `none` **and** confidence ≥ 0.7 **and** feature demand proven → the pipeline builds it unattended;
  - `needs-human` (+ the matching `risk:*`), owner assigned, and a **RUNBOOK Decision Comment** with lettered options and a recommendation. The question asked matches *why* it is held: risk kind, unproven demand, or low confidence — one genuine question, never invented ones.
- **Dedup** (plan §B.3): embedding cosine against the open clusters, falling back to `content_framework.text_similarity` (the existing post-uniqueness token-overlap measure) whenever an embedding is missing — so dedup degrades to *lexical*, never to "everything is new". A match leaves a **+1 comment with the new detail and the demand count** on the existing issue instead of filing. The classifier's own `duplicate_of` verdict is honored even below the cosine threshold.
- **Abuse guards**: a per-user 24h cap checked **before** the LLM call (the row is *held*, not dropped, and a later pass with a fresh window picks it up); a *feature* cluster needs ≥ 2 distinct reporters before it may carry `agent:ready` — bugs auto-file on the first report.
- **Privacy**: only the classifier's factual `summary` reaches GitHub — never raw report text or reporter identity. Issues correlate by internal `feedback.id`.
- **Fail-safe**: a GitHub outage returns `error` and leaves the row unclustered for the next pass, so a report is never lost. **With no token configured nothing is ever filed** — that is the kill switch.

### Clustering with no schema change

A cluster id **is** the `feedback.id` of its seed row (the first report that got an issue). New db.py helpers: `get_feedback_by_id`, `get_unprocessed_feedback` (status-filtered), `get_open_feedback_clusters` (self-join giving the seed body/embedding + `item_count`/`reporter_count`), `count_feedback_filed_by_user`, `update_feedback_triage` (writes only the fields passed). No migration.

### Beat entries

| Entry | Cadence | Does |
|---|---|---|
| `file-feedback-issues` | every 30 min | classify → dedup → file; clusters loaded once and grown in memory, so a burst of identical reports = 1 issue + N-1 `+1`s |
| `recluster-feedback` | nightly 2:45 UTC | backfills missing embeddings, attaches leftover/untriaged rows to a cluster they now match. **Files nothing** — so a nightly re-run can never open a second issue for a known problem |

Both are `QueueOnce`. Auto-closing resolved clusters is deliberately left to the changelog/notify stage (#502).

Also adds the `lem-embedding` alias (`text-embedding-3-small`) to `.litellm/config.yaml` — callers tolerate it being absent — plus the new env vars in `.env.example` and the alias row in `CLAUDE.md`.

## Testing notes

- `tests/unit/utilities/test_feedback_issue_service.py` — 123 tests, **100% coverage of the new module**: cosine/fallback similarity edge cases, dedup selection (best match, below-threshold, no-issue clusters, env/arg threshold), label logic (every emitted label asserted against the real `gh label list` set, and `agent:ready` XOR `needs-human`), MODE=start body shape/ordering/provenance, Decision Comment shape per hold reason, GitHub I/O (auth header, payload, 4xx, network error, no-token), and the orchestrator for all seven outcomes.
- `tests/unit/utilities/test_db_feedback.py` — 12 new tests for the db helpers, incl. that unknown status values are rejected without touching the DB.
- `tests/unit/app/test_feedback_issue_tasks.py` — the two tasks + both beat entries resolve to real task attributes.
- Full unit suite: **4320 passed**.

Closes #498

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: src/cqc_lem/utilities/feedback/issue_service.py [hunk 1]
```diff
@@ -0,0 +1,707 @@
...
+        return _result(IssueAction.NEEDS_HUMAN, feedback_id,
+                       confidence=classification.confidence)
+
+    vector = as_vector(row.get("embedding")) or embed_text(body)
+
+    match, score = find_duplicate_cluster(body, vector, open_clusters)
+    match = match or cluster_by_id(open_clusters, classification.duplicate_of)
+    if match:
+        reporter_count = int(match.get("reporter_count") or 1) + (1 if user_id is not None else 0)
+        item_count = int(match.get("item_count") or 1) + 1
```
[the remark this answers is not in the document]
Comment (gitchrisqueen) [reply src/cqc_lem/utilities/feedback/issue_service.py]: Good catch — fixed in 54ec2dd.

`reporter_count` is `COUNT(DISTINCT m.user_id)` over non-null user_ids, so an anonymous report is a *report* with no attributable *reporter*. Changes:
- dedup path: `int(match.get("reporter_count") or 0)` (was `or 1`), so a 0-reporter cluster stays at 0 and only gains +1 for an identified user.
- new-issue path: `reporter_count = 1 if user_id is not None else 0` (was hardcoded `1`) — this was the same inflation and it also disagreed with the cluster dict returned for in-batch dedup, which already used the 0-for-anonymous rule.
- the human-facing demand lines in `build_issue_body`/`build_duplicate_comment` now floor at 0 instead of 1, so "0 distinct user(s) across 1 feedback item(s)" is what an anonymous seed actually reads as.
- `item_count` also switched to `or 0` + 1 (a missing key used to yield 2); its display floor stays at 1 since there is always at least the current report.

New unit tests cover the anonymous seed, dedup onto a 0-reporter cluster (identified and anonymous), and that an anonymous FEATURE request does not clear the demand gate at `FEEDBACK_FEATURE_DEMAND_MIN=1` while one identified reporter does.

**Comment by gitchrisqueen** [comment]
✅ CI green, Copilot review addressed & resolved — merging (fast path).
